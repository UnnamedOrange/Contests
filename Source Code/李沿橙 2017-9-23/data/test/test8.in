5
50
863 352 670 981 946 514 258 476 36 950 550 47 606 317 404 784 427 577 675 167 389 2 103 343 993 638 437 266 334 454 533 848 3 279 561 460 481 718 581 895 759 832 920 585 629 217 170 300 60 747 
6
1 2 2 3 3 4 
50
585 165 796 126 829 406 478 414 73 405 960 801 444 733 535 77 967 217 76 950 219 326 119 285 637 486 402 589 329 670 269 113 982 771 501 617 625 864 603 375 75 890 410 525 447 8 505 538 919 981 
50
59 85 91 133 138 142 237 246 367 414 442 442 444 448 468 508 521 539 545 552 577 595 630 635 651 675 698 719 772 790 790 799 813 817 824 848 853 865 874 878 886 910 927 928 934 936 950 954 957 991 
10
1 2 3 4 5 6 7 8 9 10 
