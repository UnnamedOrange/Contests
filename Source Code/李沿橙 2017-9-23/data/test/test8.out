2 3 36 47 60 167 217 258 279 300 317 334 343 389 404 427 437 460 476 481 514 533 550 577 581 585 606 629 670 675 747 759 832 848 895 920 946 981 993 950 863 784 718 638 561 454 352 266 170 103 
1 2 3 4 3 2 
8 73 75 76 113 119 126 217 269 326 329 402 405 406 410 414 444 478 486 501 505 525 535 585 589 603 617 625 670 771 796 801 864 890 950 960 967 981 982 919 829 733 637 538 447 375 285 219 165 77 
59 85 91 133 138 237 367 442 442 444 448 468 508 521 539 545 552 595 630 635 651 675 698 772 790 790 799 813 817 824 848 853 865 874 878 910 927 928 934 936 950 954 957 991 886 719 577 414 246 142 
1 2 4 6 8 10 9 7 5 3 
